app/components/hooks/DisplayName/useDisplayName.ts

```diff
@@ -56,6 +57,25 @@ export type DisplayName =
       name: string;
     };
 
+function getVariant({
+  name,
+  accountName,
+}: {
+  name?: string;
+  accountName?: string;
+}) {
+  if (accountName) {
+    // Consider accountName as a saved name since NameController is not implemented yet
```

Comment:
it might have just been me taking an extra minute to understand the context of the saved variant. maybe 
```suggestion
    // Consider accountName as a saved name while NameController has not yet been implemented
```

Reply:
I will address this suggestion on upcoming PR to not lose your approval for now.